docs/en/observability/synthetics-reference/lightweight-config/common.asciidoc

```diff
@@ -0,0 +1,292 @@
+:hardbreaks-option:
+
+[%header]
+|===
+| Option (type) | Description
+
+// type
+| [[monitor-type]] *`type`*
+(`"http"` \| `"icmp"` \| `"tcp"`)
+a| *Required*. The type of monitor to run. One of:
+
+* `http`: Connects via HTTP and optionally verifies that the host returns the expected response.
+* `icmp`: Uses an ICMP (v4 and v6) Echo Request to ping the configured hosts. Requires special permissions or root access.
+* `tcp`: Connects via TCP and optionally verifies the endpoint by sending and/or receiving a custom payload.
+
+////////////////////////
+id
+////////////////////////
+| [[monitor-id]] *`id`*
+(<<synthetics-lightweight-data-string,string>>)
+a| *Required*. A unique identifier for this configuration. This should not change with edits to the monitor configuration regardless of changes to any config fields.
+
+*Examples*:
+
+[source,yaml]
+id: uploader-service
+
+[source,yaml]
+id: http://example.net
+
+[NOTE]
+====
+When querying against indexed monitor data this is the field you will be aggregating with. Appears in the exported fields as `monitor.id`.
+
+If you do not set this explicitly the monitor's config will be hashed and a generated value used. This value will change with any options change to this monitor making aggregations over time between changes impossible. For this reason it is recommended that you set this manually.
+====
+
+////////////////////////
+name
+////////////////////////
+| [[monitor-name]] *`name`*
+(<<synthetics-lightweight-data-string,string>>)
+a| Human readable name for this monitor.
+
+*Examples*: 
+
+[source,yaml]
+name: Uploader service
+
+[source,yaml]
+name: Example website
+
+////////////////////////
+service.name
+////////////////////////
+| [[monitor-service_name]] *`service.name`*
+(<<synthetics-lightweight-data-string,string>>)
+a| APM service name for this monitor. Corresponds to the `service.name` ECS field. Set this when monitoring an app that is also using APM to enable integrations between Uptime and APM data in Kibana.
+
+////////////////////////
+enabled
+////////////////////////
+| [[monitor-enabled]] *`enabled`*
+(<<synthetics-lightweight-data-bool,boolean>>)
+a| Whether the module is enabled.
+
+*Default*: `true`
+
+*Example*:
+
+[source,yaml]
+enabled: false
+
+////////////////////////
+schedule
+////////////////////////
+| [[monitor-schedule]] *`schedule`*
+(<<synthetics-lightweight-data-duration,duration>>)
+a| *Required*. The task schedule.
+
+*Example*:
+Run the task every 5 seconds from the time the monitor was started.
+
+[source,yaml]
+schedule: @every 5s
```

Comment:
Preview: [`schedule`](https://observability-docs_2729.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-lightweight.html#monitor-schedule)

I _think_ the format for the duration data type might be stricter in projects than it was in Heartbeat. Is that true?

I remember seeing these error messages on [the format](https://github.com/elastic/synthetics/blob/498b498eb46b9cc8d60caa880491221f06ef50d6/src/push/monitor.ts#L249) and [schedules that are less than 60 seconds](https://github.com/elastic/synthetics/blob/498b498eb46b9cc8d60caa880491221f06ef50d6/src/push/monitor.ts#L175). Is this articulated somewhere else? Should these constraints be added to the docs?  The Heartbeat docs use `@every 5s` as an example. Is that even a valid `schedule` when using projects?

Reply:
CC @shahzad31 @vigneshshanmugam is that the case? I wouldn't be surprised, since in go we use the built-in duration parsing, but it had to be emulated in JS here.

If we can get an authoritative answer here that'd be helpful.

Reply:
We do enforce the user cannot go below the 1m monitor schedule in the UI, plus saving from the Synthetics CLI spits out a warning and converts everything to the nearest supported schedule. Ex: `@every5s` would be changed to `@every60s` when pushed from the CLI.